fuel-indexer/util/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+use std::path::PathBuf;
+use std::process::Command;
+
+pub fn generate_abi(contract: &str) {
+    let base = std::env::var("CARGO_MANIFEST_DIR").unwrap();
+    let mut abi_path = PathBuf::from(base);
+
+    let out = std::env::var("OUT_DIR").unwrap();
+    let mut out_dir = PathBuf::from(out);
+
+    abi_path.push(contract);
+    out_dir.push("abi_code.rs");
+
+    Command::new("fuels-abi-cli")
```

Comment:
Is there a lib form of the abi generation code used by `fuels-abi-cli`? Would that be more friendly than requiring a system-wide installation of this cli tool?

Reply:
Yeah, I guess I could add the code to the proc macro too.